Bretonnians.cat

```diff
@@ -130,26 +130,20 @@ Double handed: S+1 AP1 Requires Two Hands</characteristic>
     </profile>
     <profile name="Burning Braziers" hidden="false" id="3e5f-babb-4f9b-76c3" typeId="c1ac-c1c8-f9d5-9673" typeName="Special Rule" page="93" publicationId="8b8d-8fc4-559e-87b1">
       <characteristics>
-        <characteristic name="Description" typeId="9f84-4221-785a-db50">The longbows of a unit equipped with burning braziers gain the Flamming Attacks special rule.</characteristic>
+        <characteristic name="Description" typeId="9f84-4221-785a-db50">The longbows of a unit equipped with burning braziers gain the Flaming Attacks special rule.</characteristic>
       </characteristics>
     </profile>
     <profile name="Defensive Stakes" hidden="false" id="642d-3ed8-534e-3e52" typeId="c1ac-c1c8-f9d5-9673" typeName="Special Rule" page="93" publicationId="8b8d-8fc4-559e-87b1">
       <characteristics>
-        <characteristic name="Description" typeId="9f84-4221-785a-db50">When a unit equipped with Defensive Stakes is deployed, the stakes are place in base contact with its front arc and will remain there for the duration of the game, unless the unit moves - should the unit move for any reason (including reforming), the stakes are lost and are removed from play. All measurements to and from the unit is done from the unit itself, ignore the stakes.
-
-
-Enemy units can charge the front of a unit equipped with Defensive Stakes as normal but do not have to physically  cross the stakes to do so. Instead, the front rank of a charging unit moves into base contact with the stakes, making a disordered charge and becoming Disrupted. A model whose troop type is &apos;cavalry&apos; or &apos;chariot&apos; must make a Dangerous Terrain test if it ends its charge move in base contact with the stakes.</characteristic>
+        <characteristic name="Description" typeId="9f84-4221-785a-db50">When a unit equipped with defensive stakes is deployed, the stakes are placed in base contact with its front arc and will remain there for the duration of the game, unless the unit moves – should the unit move for any reason (including reforming), the stakes are lost and are removed from play. All measurement to and from the unit is done from the unit itself – ignore the stakes.
+Enemy units can charge the front of a unit equipped with defensive stakes as normal but do not have to physically cross the stakes to do so. Instead, the front rank of a charging unit moves into base contact with the stakes, making a disordered charge and becoming Disrupted. A model whose troop type is ‘cavalry’ or ‘chariot’ must make a Dangerous Terrain test if it ends its charge move in base contact with the stakes.</characteristic>
```

Comment:
Thoughts on these alternative apostrophes? I can work on  getting the script to replace them, or re-saving in the editor might fix some.  I know there's some NBSP in there as well. 
Or we can leave them.

Reply:
I prefer standard apostrophes, alternative ones sometimes cause problems in PDF export among other things